android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -42,13 +50,10 @@
 import android.graphics.Rect;
 import android.net.Uri;
 import android.os.Build;
-import android.view.MotionEvent;
-import android.view.View;
 import android.webkit.WebSettings;
 import android.webkit.WebView;
 
-@SuppressWarnings("deprecation")
-public class TiUIWebView extends TiUIView
+@SuppressWarnings("deprecation") public class TiUIWebView extends TiUIView
```

Comment:
Yep. The file I imported have those two lines. As for how I imported them: Preferences->Editor->Code Style->Java then import eclipse xml and select. For reformatting, cmd + a then code -> reformat code. Still same result.